public/i18n/it-it.json

```diff
@@ -24,6 +24,8 @@
   "ftm-filters" : "Filtri",
   "ftm-france" : "Francia",
   "ftm-french" : "francese",
+  "ftm-italian" : "italiano",
+  "ftm-german" : "Tedesco",
```

Comment:
Tiny nit: inconsistent capitalization across language names.

Reply:
this is google MT, I'll update

Reply:
Going to merge as is but I edited it in Smartling so it'll come through soon